frontend/packages/api_client/src/models/OpenEdXInstanceConfig.ts

```diff
@@ -70,14 +70,6 @@ export interface OpenEdXInstanceConfig {
      * @memberof OpenEdXInstanceConfig
      */
     instanceName: string;
-    /**
-     * The email your instance of Open edX will be using to send emails, and where your users should send their support requests.
-     * 
-     * This needs to be a valid email.
-     * @type {string}
-     * @memberof OpenEdXInstanceConfig
-     */
-    publicContactEmail: string;
```

Comment:
@rusrushal13 You should not remove `publicContactEmail` from the API endpoint, just make it an optional parameter and auto-fill it with the user's email if it's not filled in.

Reply:
Users should be able to edit this field in the console, we just want to remove it on the registration form.

Reply:
Thanks @giovannicimolin , I have updated the PR :)